src/ocr/ppocr/ppocrv5.cpp

```diff
@@ -309,11 +309,11 @@ void PaddleOCRAppV5::rec(const std::vector<cv::Mat> &detectImg)
         input.substract_mean_normalize(mean_vals, norm_vals);
         auto outIndexes = recNet->output_indexes();
         ncnn::Extractor extractor = recNet->create_extractor();
-        if (recNet->opt.use_vulkan_compute) {
```

Comment:
It is recommended to make two commits. Regarding OCR, it is not recommended to delete it. If you want to delete it, you can modify this separate commit.